src/digitalmarketplace/components/list-input/list-input.js

```diff
@@ -0,0 +1,211 @@
+var getSibling = function (direction, elem, selector) {
+  // Get the next sibling element
+  var sibling = (direction === 'next') ? elem.nextElementSibling : elem.previousElementSibling
+
+  // If there's no selector, return the first sibling
+  if (!selector) return sibling
+
+  // If the sibling matches our selector, use it
+  // If not, jump to the next sibling and continue the loop
+
+  // For IE10,11 (Polyfill for matches)
+  if (!Element.prototype.matches) {
+    Element.prototype.matches = Element.prototype.msMatchesSelector ||
+                                Element.prototype.webkitMatchesSelector
+  }
+  while (sibling) {
+    if (sibling.matches(selector)) return sibling
+    sibling = (direction === 'next') ? sibling.nextElementSibling : sibling = sibling.previousElementSibling
+  }
+}
+
+function ListInput ($module) {
+  this.$module = $module
+  this.$allVisibleItems = $module.querySelectorAll('.dm-list-input__item:not(.dm-list-input__item--hidden)')
```

Comment:
is this use of `:not` safe?

Reply:
🤦  I promise I'm not missing these on purpose.